modules/Bio/EnsEMBL/Compara/Utils/FlatFile.pm

```diff
@@ -73,4 +74,28 @@ sub map_row_to_header {
     return $row;
 }
 
+=head2 parse_flatfile_into_hash
+
+    A two column file is parsed into $column_1->$column_2
+
+=cut
+
+sub parse_flatfile_into_hash {
+    my ($self, $filename, $filter) = @_;
+
+    my %flatfile_hash;
+    open(my $fh, '<', $filename) or die "Cannot open $filename for reading";
+    my $header = <$fh>;
+    while ( my $line = <$fh> ) {
+        chomp $line;
+        my ( $id, $val ) = split(/\s+/, $line);
+        next if $val eq '';
+        next if $filter && ! $self->match_range_filter($id, $filter);
```

Comment:
```suggestion
        next if $filter && ! $self->_match_range_filter($id, $filter);
```
If this was just a copy-paste, that underscore is missing. If not, there is no `match_range_filter` in Compara's codebase. Where does it come from?

Reply:
Nope, none of the above, a missed reversal of a previous change